src/test/resources/pages/adminEmailCompose.html

```diff
@@ -27,7 +27,7 @@
          </script>
          <script src="/tinymce/js/tinymce/tinymce.min.js" type="text/javascript">
          </script>
-         <link href="http://localhost:8888/tinymce/js/tinymce/skins/lightgray/skin.min.css" rel="stylesheet"/>
```

Comment:
I was wrong as well. In fact, they both have to be different for God mode to work because there are two different URLs to be used. Perhaps this should be checked as well along with the `areTestAccountDefaultValues()`?
So yeah certainly the explicit localhost:8888 replace is not needed. Rather it should be `TestProperties.inst().TEAMMATES_URL` to be replaced with `${web.url}` or whatever variable name, but I'm pretty sure you've figured this out already :-)